### PR TITLE
started off the About us page

### PR DESCRIPTION
**Pull request by idjevm**
### What does this pull request aim to achieve? 💯
This is a starting point for the About Us page. I am waiting on the assets for this page and will work on the design as well. 
_Please be concise and link any related issue(s):_
#589

### CODE REVIEW

**Comment by galeza** [comment]
Hi @idjevm 
I compared  https://featurecreate-about-us-page.fightpandemics.xyz/about-us to the zeplin design.
- Are we going to add two buttons below youtube video: Join as a volunteer and join as an Ambassador?
- One spelling mistake: "anyone involved he ould not ask" -> anyone involved he **could** not ask

**Comment by idjevm** [comment]
> Hi @idjevm
> I compared https://featurecreate-about-us-page.fightpandemics.xyz/about-us to the zeplin design.
> 
> * Are we going to add two buttons below youtube video: Join as a volunteer and join as an Ambassador?
> * One spelling mistake: "anyone involved he ould not ask" -> anyone involved he **could** not ask

Yeah, we removed that section all together, maybe will add another time. And this should be good to go :D @galeza  so let me know if you see something not good.

**Comment by galeza** [comment]
lgtm

File: client/src/routes.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import Home from "./pages/Home";
 import NeedHelp from "./pages/NeedHelp";
 import OfferHelp from "./pages/OfferHelp";
-import About from "./pages/About";
+import AboutUs from "./pages/AboutUs";
```
Comment (Joeu) [inline client/src/routes.js:4]: Later we could have an _index.js_ at the _/pages_ exporting all the pages, and then, in the routes, import them like this:

`
import {
  Home,
  NeedHelp,
  OfferHelp,
  AboutUs
} from "./pages";
`

Comment (idjevm) [reply client/src/routes.js:4]: Noted! I think there will be plenty of refactoring to be done.

**Pull request merged**